common/source/docs/common-buttons.rst

```diff
@@ -4,23 +4,51 @@
 Buttons
 =======
 
-Up to four button inputs to the autopilot may be defined that will execute an :ref:`Auxiliary Function <common-auxiliary-functions>` as if an RC channel switch had been used to execute it. The button input can be either a switch to ground, or a PWM input signal.
+Up to four external buttons or switches connected to the autopilot can be configured to trigger :ref:`Auxiliary Functions <common-auxiliary-functions>` similar to how RC channel switches can trigger them. These button inputs can be either analog voltage inputs (pull-to-ground or pull-to-high) or PWM inputs.
```

Comment:
Txs, I've updated the top paragraph.  I've left the bottom paragraph out for now though because the parameter descriptions don't include pin numbers for anything except the AUX OUT pins.

Reply:
those are given as an aid  for IOMCU based autopilots...any appropriate pin can be input....if that is confusing we should change the value and descriptions to make it clear....those numbers for a MATEK flight controller are SERVO1-SERVO6, etc.